examples/cluster/playground/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM docker-dev.yelpcorp.com/xenial_yelp
```

Comment:
as this is public repo, can we use ubuntu:bionic here? it should have python3.6 packages